get_flight/src/get_flight_url_utils.py

```diff
@@ -0,0 +1,72 @@
+from decimal import *
+from datetime import date, timedelta, datetime
+import boto3
+
+dynamodb = boto3.resource('dynamodb')
+
+
+def make_flight_url(origin, destination, departure_date, return_date):
+    url = ("https://www.skyscanner.de/transport/fluge/"+ origin+"/"+ destination+"/"+departure_date+"/"+return_date+"/")
+    return url
+
+def startdate():
+    return date.today() + timedelta(days =1)
+
+def returndate(start_date, trip_length):
+    return start_date + timedelta(days = trip_length)
+
+def maketimestring(day):
+    return day.strftime("%y%m%d")
+    return date
+
+def get_all_destinations_from_dynamodb():
+    table_destination = dynamodb.Table('destination')
+    response = table_destination.scan()
+    data = response['Items']
+    while 'LastEvaluatedKey' in response:
+        response = table_destination.scan(ExclusiveStartKey=response['LastEvaluatedKey'])
+        data.extend(response['Items']) 
+    return data 
+
+
+def write_write_url_to_dynamo(city, city_id, url):
+    table_destination = dynamodb.Table('destination')
```

Comment:
you define the multiple times in this file.
maybe move name or table to constant or